qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider.h

```diff
@@ -294,40 +294,34 @@ class RosPluginlibPluginProvider
 
     std::string manifest_path = class_loader_->getPluginManifestPath(lookup_name);
     //qDebug("RosPluginlibPluginProvider::parseManifest() manifest_path \"%s\"", manifest_path.c_str());
-    TiXmlDocument doc;
-    bool loaded = doc.LoadFile(manifest_path);
-    if (!loaded)
+    tinyxml2::XMLDocument doc;
+    bool loaded = doc.LoadFile(manifest_path.c_str());
+    if (!loaded || doc.Error())
     {
-      if (doc.ErrorRow() > 0)
-      {
-        qWarning("RosPluginlibPluginProvider::parseManifest() could not load manifest \"%s\" (%s [line %d, column %d])", manifest_path.c_str(), doc.ErrorDesc(), doc.ErrorRow(), doc.ErrorCol());
-      }
-      else
-      {
-        qWarning("RosPluginlibPluginProvider::parseManifest() could not load manifest \"%s\" (%s)", manifest_path.c_str(), doc.ErrorDesc());
-      }
+      qWarning("RosPluginlibPluginProvider::parseManifest() could not load manifest \"%s\" (%s)", manifest_path.c_str(), doc.ErrorStr());
```

Comment:
Is the line / column information not available in TinyXML2 anymore?

Reply:
Unfortunately, TinyXML2 cannot report the row/column of parse errors

http://www.grinninglizard.com/tinyxml2/